MAINTAINERS

```diff
@@ -124,8 +124,90 @@ the relevant operators.
 
 * If the change affects the governance, philosophy, goals or principles of the project,
 it must be approved by BDFL.
+
+* A pull request can be in 1 of 5 distinct states, for each of which there is a corresponding label
+that needs to be applied. `Rules.review.states` contains the list of states with possible targets
+for each.
 """
 
+		# Triage
+		[Rules.review.states.nolabel]
+
+			# Maintainers are expected to triage new incoming pull requests by adding
+			# the correct labels (e.g. `1-design-review`) potentially skipping some steps
+			# depending on the kind of pull request. Use common sense for judging.
+			#
+			# Checking for DCO should be done at this stage.
+			#
+			# If an owner, responsible for closing or merging, can be assigned to the PR,
+			# the better.
+
+			close = "unresponsive contributor without DCO"
+			3-docs-review = "Non-proposal documentation-only change"
+			2-code-review = "trivial bugfix"
+			1-design-review = "general case"
+
+		# Design review
+		[Rules.review.states.1-design-review]
+
+			# Maintainers are expected to comment on the design of the pull request.
+			# Review of documentation is expected only in the context of design validation,
+			# not for stylistic changes.
+			#
+			# Ideally, documentation should reflect the expected behavior of the code.
+			# No code review should take place in this step.
+			#
+			# Once design is approved, this label should be removed and the next label
```

Comment:
Per @crosbymichael 's suggestion, we should be more specific here on maintainers' duty wrt to the workflow.

I suggest we change this to `Once design is approved, a maintainer (the owner, if there is), should remove the current label and add the next label.`